frontend/src/toolbar/elements/elementsLogic.ts

```diff
@@ -383,6 +388,21 @@ export const elementsLogic = kea<
             } else {
                 actions.setSelectedElement(element)
             }
+
+            posthog.capture('viewed toolbar element', {
```

Comment:
Should this be `clicked` or `selected` instead of viewed?

Reply:
good suggestion! updated it to `toolbar selected HTML element` so that it properly reflects the action, starts with the keyword `toolbar` and makes it very, very clear that you're selecting an element in the host's page.